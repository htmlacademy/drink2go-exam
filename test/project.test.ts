import { describe, expect, it, test } from 'vitest'
import { lint } from 'stylelint'
describe('project-good test', () => {

  it('bar', () => {
    expect(1 + 1).eq(2)
  })

  it('snapshot', ({ expect }) => {
    expect({ foo: 'bar' }).toMatchSnapshot()
  })

  it('style lint', async ({ expect}) => {
    const { errored } = await lint({
      config: {
        extends: 'stylelint-config-htmlacademy'
      },
      files: 'project/**/*.scss'
    })
    expect(errored).toBeFalsy()
  })
})

