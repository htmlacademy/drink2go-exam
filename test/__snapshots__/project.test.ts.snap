// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`project test > snapshot 1`] = `
{
  "foo": "bar",
}
`;

exports[`suite name > snapshot 1`] = `
{
  "foo": "bar",
}
`;
